src/ios/THEPeer.m

```diff
@@ -38,7 +38,7 @@ @implementation THEPeer
 }
 
 // Class initializer.
-- (instancetype)initWithIdentifier:(NSUUID *)identifier
+- (instancetype)initWithIdentifier:(NSString *)identifier
                               name:(NSString *)name
```

Comment:
If you consider the class in isolation, maybe yes. But in fact it's only ever called in response to finding a peer which is guaranteed to produce a peerIdentifier.